### PR TITLE
Health design book

### PR DESCRIPTION
**Pull request by ericbenwa**

### CODE REVIEW

**Comment by craigmcginley** [comment]
Deploy preview for *loving-austin-5f2eff* ready!

Built with commit 5e20ce00d24ca4f6c0497b7fe7a8d474df4ac6d6

https://deploy-preview-277--loving-austin-5f2eff.netlify.com

**Comment by ericbenwa** [comment]
@jsonin Take a peek!

File: src/pages/vision/healthcare-dollars/npm-debug.log [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+8 error argv "/Users/ericbenoit/.nvm/versions/node/v6.11.5/bin/node" "/Users/ericbenoit/.nvm/versions/node/v6.11.5/bin/npm" "config" "--loglevel=warn" "get" "prefix"
+9 error node v6.11.5
+10 error npm  v3.10.10
+11 error code EPIPE
+12 error errno EPIPE
+13 error syscall write
+14 error write EPIPE
+15 error If you need help, you may report this error at:
+15 error     <https://github.com/npm/npm/issues>
+16 verbose exit [ 1, true ]
```
Comment (craigmcginley) [inline src/pages/vision/healthcare-dollars/npm-debug.log]: Should delete this and maybe add *.log to gitignore. Make sure you use yarn too there might be discrepancies if not.

File: src/data/features.json [hunk 2]
```diff
@@ -1,4 +1,15 @@
...
+    "id": "health-design-thinking",
+    "title": "Health Design Thinking Book",
+    "date": "Mar.2020",
+    "client": "Feature",
+    "categories": [],
+    "caption": "GoInvo highlighted in Health Design Thinking Book",
+    "image": "/images/features/health-design-thinking/health-design-thinking-book-hero-5.jpg",
+    "link": "/vision/health-design-thinking/",
+    "hiddenWorkPage": true
+  },
```
Comment (craigmcginley) [inline src/data/features.json]: Is this meant to display on the Vision page or is it more of a one off promo for the event?

File: src/pages/vision/health-design-thinking/index.js [hunk 3]
```diff
@@ -0,0 +1,235 @@
...
+import Hero from '../../../components/hero'
+import ImageBlock from '../../../components/image-block'
+import Quote from '../../../components/quote'
+import HubspotForm from '../../../components/hubspot-form'
+import Image from '../../../components/image'
+import Divider from '../../../components/divider'
+import Columns from '../../../components/columns'
+import Card from '../../../components/card'
+
+import config from '../../../../config'
```
Comment (craigmcginley) [inline src/pages/vision/health-design-thinking/index.js]: Just check your terminal for warnings coming from this file and clean those up. I think there are a couple unused imports, for instance. Any warnings you didn't create with this file you can ignore. Thanks.

File: src/data/features.json (hunk 2 above)
Comment (ericbenwa) [reply src/data/features.json]: Meant to display on Vision page.
We'll want to make people aware of this even past launch.
Though we may want to consider a different layout for the Vision landing page.

File: src/pages/vision/healthcare-dollars/npm-debug.log (hunk 1 above)
Comment (ericbenwa) [reply src/pages/vision/healthcare-dollars/npm-debug.log]: I'm not using Yarn?
I'm doing that unknowingly... show me the way please!

File: src/pages/vision/health-design-thinking/index.js (hunk 3 above)
Comment (ericbenwa) [reply src/pages/vision/health-design-thinking/index.js]: Good find, thanks!
Wasn't aware of those clues in Terminal... and now I know!
https://www.youtube.com/watch?v=pele5vptVgc

**Comment by ericbenwa** [comment]
Please take a peek and see if I corrected those issues properly... thanks!

**Pull request merged**